Packages/com.unity.inputsystem/InputSystem/Unity.InputSystem.asmdef

```diff
@@ -111,6 +111,11 @@
             "name": "com.unity.modules.unityanalytics",
             "expression": "1",
             "define": "UNITY_INPUT_SYSTEM_ENABLE_ANALYTICS"
+        },
+        {
+            "name": "Unity",
+            "expression": "6000.2.0a11",
```

Comment:
Note that this is only to allow this PR to be tested with the Unity branch (internal). It would have to be updated to the Unity version containing the actual update to module before this PR could land.

Reply:
If you are testing this branch consider updating this to whatever Unity version number the associated Unity branch has.